### ISSUE TITLE
Training schedule about VOC.

### ISSUE THREAD

**Issue by Retiina** [google-research/ssl_detection#18]
Dear sir, really thanks to your great work! But I am stuck at the detail about training schedule of VOC stage1 now. May I ask some questions:
1. The Batch size is 1 and total GPU is 1.
2. Lr schedule is [7500, 10000], so total epoch is 10000 * 8 / 5011 = 16 epoch
3. learning rate is 0.001, decay rate is 0.5
Thanks in advance!
